app/src/test/java/com/main/mealplanner/RecipeUnitTests.kt

```diff
@@ -73,6 +73,11 @@ class RecipeUnitTests {
         mvm.fetchRecipe("aslkjgaoh")
         thenIGetNoResults()
     }
+
+    private fun givenAFeedOfMockedRecipeDataAvailable() {
+        TODO("Not yet implemented")
+    }
+
```

Comment:
Created a function that not yet created that being called from
" fun searchForGarbage_returnsNothing() "
and that fix one of the errors

Reply:
I'd leave this out... an unimplemented function is essentially clutter, which is tech debt.